tests/phpunit.xml

```diff
@@ -1,4 +1,9 @@
 <phpunit bootstrap="bootstrap.php">
+    <groups>
+        <exclude>
```

Comment:
I'm still figuring why the base suite currently fails. I will update travis config as soon as the pre-existing suite is passing again.